.github/workflows/e2e.yml

```diff
@@ -49,6 +49,12 @@ jobs:
       - name: Build create-llama
         run: pnpm run build
         working-directory: ./packages/create-llama
+      - name: Pack
+        run: pnpm pack
+        working-directory: ./packages/create-llama
+      - name: Extract Pack
+        run: tar -xvzf create-llama-0.0.21.tgz
```

Comment:
1. that will break if we update the version
2. better use temp dir
(see test.yml)